app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java

```diff
@@ -247,17 +250,17 @@ public void onCancelled(@NonNull DatabaseError error) {
     @Override
     protected void onDestroy() {
```

Comment:
This method is really hard to understand, maybe add comments and simplifly it. For example, add block
`if(thisGame == null){ return; }` instead of always checking the negation and creating a new block.

Reply:
I agree that it is hard to understand, I don't really know what it does because I didn't code it